.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--- Describe your changes in detail -->
+What component does this PR affect?
+<!--- Write here -->
+Requires changes from other components?
+
+(Yes/No)
+
+What are the changes required from other components?
+<!--- Write here -->
+
+### Related Issue
+<!--- This project only accepts pull requests related to open issues -->
```

Comment:
> This project only accepts pull requests related to open issues

Let's make this more welcoming saying that please make sure you have an issue associated with the pull request.

Also, add another question to add `(close/fix #<issue-no>)` to the pull request title